README.md

```diff
@@ -8,6 +8,8 @@ as developed by the [JSON for Linking Data Community Group](https://www.w3.org/c
 
 The Use Cases and Requirements document can also be read  [read directly](https://json-ld.github.io/yaml-ld/UCR/).
 
+The YAML-LD Namespace contains a [convenience context](https://www.w3.org/ns/yaml-ld/v1) which can also be [red directly](https://json-ld.github.io/yaml-ld/ns/v1.html).
```

Comment:
```suggestion
The YAML-LD Namespace contains a [convenience context](https://www.w3.org/ns/yaml-ld/v1) which can also be [read directly](https://json-ld.github.io/yaml-ld/ns/v1.html).
```

Reply:
Fixed in 314b84df664448f948df3224ca27336304b90388.